faq.rst

```diff
@@ -37,10 +38,11 @@ Type
 **QUESTION**: What behavior is expected from a purl spec implementation if a
 ``type`` contains a character like a slash '/' or a colon ':'?
 
-The "Rules for each purl component" section provides that
+**ANSWER**: The "Rules for each purl component" section provides that the
+package ``type``
 
-    [t]he package ``type`` MUST be composed only of ASCII letters and numbers,
-    '.', '+' and '-' (period, plus, and dash)
+    MUST be composed only of ASCII letters and numbers, '.', '+' and '-'
```

Comment:
@johnmhoran let's use a unified approach for characters like in the encoding section e.g., `period '.'`

Reply:
Agreed and done.